packages/foreman/foreman/foreman.spec

```diff
@@ -759,6 +762,22 @@ install -Dp -m0644 %{SOURCE5} %{buildroot}%{_tmpfilesdir}/%{name}.conf
 install -Dp -m0644 extras/systemd/%{name}.service %{buildroot}%{_unitdir}/%{name}.service
 install -Dp -m0644 extras/systemd/%{name}.socket %{buildroot}%{_unitdir}/%{name}.socket
 
+# SCL libexec wrappers
+%if 0%{?scl:1}
+  cat > %{buildroot}%{_libexecdir}/%{name}/%{scl_name}-ruby-selinux <<-EOF
+  #!/bin/bash
+  # Shell wrapper for %{scl_name} with SELinux transition into foreman_rails_t domain.
+  source scl_source enable %{scl_name}
+  exec ruby "\$@"
+EOF
```

Comment:
Wouldn't this lead to all the lines being prefixed with two empty spaces and possibly messing up the shebang?